src/transactions/TransactionTable.js

```diff
@@ -2,21 +2,43 @@ import React from 'react';
 import { PropTypes } from 'prop-types';
 import { v4 as uuid } from 'uuid';
 import { useSelector } from 'react-redux';
+// import { propTypes } from 'react-bootstrap/esm/Image';
 import TransactionRow from './TransactionRow';
 import TransactionForm from './TransactionForm';
 
 /**
  * Shows a table of transactions and optionally a form for creating a transaction
- *
+*Added props to the TransactionTable component that allows filtering by:
+start date (inclusive),end date (inclusive), category, payee.
+
+// "filters" is prop name for now
+// The "filters" prop is optional,
+it may not be set at all, in which case the table should show everything.
+// "filters" prop may be missing or set to null, and those should be ignored.
+
```

Comment:
Most of these comments should be moved and consolidated to be part of the JSDoc comment for the filters prop below. Maybe a small note in here that the table can be filtered. JSDoc comments are the documentation of our code, so they should generally describe how this component works for other people using this component, not what you specifically did in this assignment.